lib/crypto.js

```diff
@@ -0,0 +1,28 @@
+// browser crypto, used in browserify instead of shimming node's crypto package
+
+(function(global, module, define) {
+
+
+function randomBytes(width) {
+  var out = new Uint8Array(width);
+  (global.crypto || global.msCrypto).getRandomValues(out);
+  return out;
+}
+
+var impl = { randomBytes: randomBytes }
+
+if (module && module.exports) {
+  module.exports = impl;
+} else if (define && define.amd) {
+  define(function() { return impl; });
+} else {
+  this.crypto = impl;
+}
+
+})(
+  this,
+  (typeof module) == 'object' && module,    // present in node.js
```

Comment:
good point. I put this in because I copied the format of all other modules in this project.  I was thinking this might be included in an amdjs scenario but I don't think it will.